dev/io.openliberty.netty.internal.impl/src/io/openliberty/netty/internal/impl/NettyFrameworkImpl.java

```diff
@@ -70,7 +74,7 @@
 public class NettyFrameworkImpl implements ServerQuiesceListener, NettyFramework {
 
     private static final TraceComponent tc = Tr.register(NettyFrameworkImpl.class, NettyConstants.NETTY_TRACE_NAME,
-            NettyConstants.BASE_BUNDLE);
+            NettyConstants.CF_BUNDLE);
```

Comment:
I need to open an issue to address this in the future if we have any Netty specific messages we want to / need to add.

Reply:
https://github.com/OpenLiberty/open-liberty/issues/31505